tests/integration/test_apply_checks.py

```diff
@@ -5001,13 +5001,20 @@ def test_apply_checks_all_checks_using_classes(ws, spark):
             user_metadata={"tag1": "value7", "tag2": "033"},
```

Comment:
Please also add this check `all_row_checks.yaml` so that execution of it is checks in streaming.